evaluation.md

```diff
@@ -14,6 +14,14 @@ describe_ why _you made the change._
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+I didn't make that many changes from my ideal syntax. I added some funcitonality, like being able to use "e", "East", "E", "east" interchangably. I thought I would need to implement some weird indexing for getting a integer from a list of states to keep track of states like Picobot does, but it worked out okay!
+The big change I had to make was, it only accepts one rule at a time! So, passing in multiple rules at a time will not work. The reason I made this change is because it is 11:22 PM, and I hate Scala, and I hate Parser Combinators.
```

Comment:
I also got frustrated while attempting to implement mine too, I found it much more difficult/time consuming than all of the other HWs